packages/cra-template-foundations/template/src/index.tsx

```diff
@@ -30,8 +30,9 @@ export const renderApp = (Component: React.ComponentType) => {
 
 const run = async () => {
   try {
+    const globalConfig: Config = config
     // Set the global config
-    window.reapit.config = config as Config
+    window.reapit.config = globalConfig
```

Comment:
do we need the delcare here?

Reply:
Not sure why but TS didn't like it unless I added the type 🤷🏻